app/models/forecast.rb

```diff
@@ -0,0 +1,30 @@
+class Forecast
+  attr_reader :city_name,
+              :country_name,
+              :sunrise_time,
+              :sunset_time,
+              :description,
+              :temp,
+              :temp_min,
+              :temp_max,
+              :pressure,
+              :humidity,
+              :visibility,
+              :wind
+
+  def initialize(forecast_info)
```

Comment:
This looks great! We'll just have to figure out how to convert sunrise and sunset time into time objects.